src/quo2/components/community/community_list_view.cljs

```diff
@@ -71,14 +71,14 @@
          {:locked? locked?
           :tokens  tokens}]
         [notification-view
-         {:muted?                muted?
+         {:muted?                muted
           :unread-mentions-count unread-mentions-count
           :unread-messages?      unread-messages?}])]]]])
 
 (defn communities-membership-list-item
   [props
    {:keys [name
-           muted?
+           muted
```

Comment:
Why change `muted?` to `muted`. Isn't that a boolean?

Reply:
@OmarBasem this is because the endpoint returns `muted` instead of `muted?`. Initially it was `muted?` because the data was mocked